tests/helper.js

```diff
@@ -220,12 +220,16 @@ module.exports = {
 
   waitElementToBeVisible: (elm, timeout) => {
     timeout = timeout || 15000;
-    browser.wait(EC.visibilityOf(elm), timeout);
+    browser.wait(EC.visibilityOf(elm), timeout, 'waitElementToBeVisible timed out looking for ' + elm.locator());
   },
 
   waitElementToBeClickable: (elm, timeout) => {
     timeout = timeout || 15000;
-    return browser.wait(EC.elementToBeClickable(elm), timeout);
+    const msg = 'waitElementToBeClickable timed out looking for ';
+    if (elm.locator()){
+      msg + elm.locator();
+    } 
```

Comment:
I had it that way originally and it would fail the test vs just making it undefined. I forget the exact error now. I'll go back and see what the message is.